.github/workflows/binaries.yml

```diff
@@ -11,9 +11,11 @@ jobs:
       - uses: actions/checkout@v4
       - id: generate
         run: |
+          export WEEDER_BINARIES_OVERWRITE=1
```

Comment:
This only affects the build script, and is used to tell it to include existing binaries in the matrix.

I'm not sure why that was useful, maybe testing, but it won't affect the `gh release upload` command later, which needs `--clobber` not to fail trying to upload an existing asset. It would probably be safe to have `--clobber` all the time.